README.md

```diff
@@ -117,26 +118,39 @@ Instead of `make`, you can use `make -jN` where `N` is your number of CPU cores
 
     ./daemon
 
-¹ *Fast, requires `aria2c`.*
+¹ *Fast, requires `aria2c`.*  
 ² *Unreliable speed, requires `curl`.*
 
-#### If you're a developer
+#### Loading base asset package from sources
+
+As a developer, you will want to load assets from repository. To do that:
+
+    cd build
+	./daemon -pakpath ../pkg -set vm.sgame.type 3 -set vm.cgame.type 3
+
+Note that only the basic `unvanquished_src.dpkdir` asset package is provided that way, and running Unvanquished only with that package will bring you some warnings about other missing packages and you will miss soundtrack and stuff like that. Note that you need to load your own game code (using _vm type_ switches) at this point.
+
+This should be enough to start the game and reach the main menu and from there, join a server. If the server supports autodownload, Unvanquished will fetch all the missing package from it.
+
+If you are looking for the sources of the whole assets, have a look at the [UnvanquishedAssets](https://github.com/UnvanquishedAssets/UnvanquishedAssets) repository. Beware, unlike the `unvanquished_src.dpkdir` package, most of them can't be loaded correctly by the engine without being built first.
+
+#### Loading your own assets
 
 As a developer, you will want to load your own assets in addition to those shipped with the game. To do that:
 
     cd build
     mkdir -p pkg; cd pkg
-    ln -s ../../main git_source.pk3dir
-    mkdir assets_source.pk3dir
-    echo git > assets_source.pk3dir/DEPS
+    mkdir assets_src.dpkdir
 
-You can now put loose assets into `assets_source.pk3dir` or you can put additional pk3dir directories or pk3 containers inside `pkg` and add their names (the format is `NAME_VERSION.pk3(dir)`) as lines to the `DEPS` file. In order to launch Unvanquished, use one of the following commands:
+You can now put loose assets into `assets_src.dpkdir` or you can put additional dpkdir directories or dpk containers inside `pkg` and add their names (the format is `<NAME>_<VERSION>.dpk[dir]`) as lines to the `DEPS` file (the format is `<NAME> <VERSION>`. Version is required in package filename but optional in `DEPS` file. In order to launch Unvanquished, use one of the following commands:
 
-  - `./daemon -pakpath PATH -set fs_extrapaks assets # ¹`
-  - `./daemon -pakpath PATH -set fs_extrapaks assets -set vm.sgame.type 3 -set vm.cgame.type 3 -set vm.sgame.debug 1 -set vm.cgame.debug 1 +devmap plat23 # ²`
+  - `./daemon -set fs_extrapaks assets # ¹`
+  - `./daemon -pakpath PATH -set fs_extrapaks assets # ²`
+  - `./daemon -pakpath PATH -set fs_extrapaks assets -set vm.sgame.type 3 -set vm.cgame.type 3 -set vm.sgame.debug 1 -set vm.cgame.debug 1 +devmap plat23 # ³`
```

Comment:
The `debug` VM flags have no effect when the type is 3 (DLL). Since in that case, one simply attaches the debugger to `daemon`.